doc/source/api.rst

```diff
@@ -8,6 +8,7 @@ API Reference
    :param df: Data to be written out.
    :param database: Name / location of the Hyper file to write to.
    :param table: Table to write to. Must be supplied as a keyword argument.
+   :param hyper: A `HyperProcess` in case you want to spawn it by yourself. Optional. Must be supplied as a keyword argument.
```

Comment:
Minor nit but I think `hyper_process` or `process` is a better parameter name. The readability of `frame_from_hyper("example.hyper", hyper=foo)` feels a little off

Reply:
fixed, here and in #117